sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/UnnamedSessionReceiver.java

```diff
@@ -75,10 +74,6 @@
         this.renewSessionLock = renewSessionLock;
         this.lockContainer = new MessageLockContainer(ServiceBusConstants.OPERATION_TIMEOUT);
 
-        final AmqpErrorContext errorContext = new LinkErrorContext(receiveLink.getHostname(),
-            receiveLink.getEntityPath(), null, null);
```

Comment:
import for `AmqpErrorContext` needs to be removed since it is not required any more.

Reply:
Yeah. Good thing spotbugs let me do it.